tutorial/06_parsing_template/run

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+parsetemplate --output student/code.py template.py
+Output=$(run_student python3 student/code.py)
```

Comment:
Maybe I'm pedant, but the common pratice in shell scripts is to write the whole variable names either in upper case, for environment variables and constants, or in lower case, for the rest of them.